.github/workflows/gradle-latest-versions.yml

```diff
@@ -0,0 +1,45 @@
+# yaml-language-server: $schema=https://json.schemastore.org/github-workflow.json
+name: Gradle latest versions tests
+
+on:
+  schedule:
+    - cron: "0 7 * * MON"
+  push:
+    branches:
+      - "gradleLatestVersionsTestManualTrigger"
+      - "szpak/gradleLatestVersionsGHActions"
+
+jobs:
+  gradle-compatTest:
+    name: "Compatibility tests with latest Gradle versions"
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-java@v3
+        with:
+          java-version: 11
+          distribution: 'zulu'
+
+      - name: Fetch latest available Gradle versions
+        run: |
+          GRADLE_VERSIONS_TYPES_TO_FETCH="current release-candidate nightly release-nightly"
+          GRADLE_VERSIONS_OUTPUT_FILE=latest-gradle-versions.txt
+
+          for gv in $GRADLE_VERSIONS_TYPES_TO_FETCH; do
+            echo "Fetching latest Gradle version for '$gv'"
+            curl --silent --show-error "https://services.gradle.org/versions/$gv" | jq -r '.version | select( . != null )' >> latest-gradle-versions.txt
```

Comment:
Isn't `gradlew stutterRefreshLocks` giving you similar results?

Reply:
Could you elaborate more? I've checked the https://github.com/ajoberstar/gradle-stutter repo, but I [searching for "refresh"](https://github.com/ajoberstar/gradle-stutter/search?q=refresh) did not return anything :-/.

Reply:
https://github.com/ajoberstar/gradle-stutter/blob/709895b0061354ef5999f32444652e9f10b7bf94/src/main/java/org/ajoberstar/gradle/stutter/StutterPlugin.java#L35-L38 ?

Reply:
Yep that, sorry.

Reply:
1. We don't use the latest version and I'm not sure if it wasn't added in 0.7.x (which [dropped support for JDK 8](https://github.com/ajoberstar/gradle-stutter/releases/tag/0.7.0)).
2. I'm not sure about the nightly version available there (but maybe we don't need it?)
3.

Reply:
1. It still has that task, that's how I did #164. My point here was that rather than manually finding the latest, just run that task and then run the builds. It'll pick up the latest Gradle versions anyway.
2. Yes nightlys won't be there, but that's not necessary anyway, because you would be exposed to potentially buggy Gradle builds resulting in flaky CI builds every Monday, which is pretty random. I wonder if you can tell Stutter to use RCs though.

Reply:
I propose to merge it as is and keep it running (the next window is ~12h), until we switch to building with [JDK 11+](https://github.com/gradle-nexus/publish-plugin/issues/171), bump the Stutter and find a way to use also RC builds.

Reply:
All yours, was just raising a potential issue.